### DISCUSSION TITLE
Error when trying to install NTFS-3g

### DISCUSSION THREAD

**Discussion by zishanj4** [Homebrew/discussions#1180] [Getting started]
Hi, im trying to install ntfs through homebrew. but i get an error saying fuse hasnt bene installed, but it is installed so im not sure why its not detecting it. 
<img width="493" alt="Screenshot 2021-04-02 at 23 49 07" src="https://user-images.githubusercontent.com/65806462/113459691-1f44fd00-940e-11eb-9abe-0e18f7468c8d.png">

**Comment by gromgit** [comment]
Please post text output _as text_, not screenshots. Images of text are truly inconvenient to deal with.

Did you do what the warning suggested?
```
brew install --cask osxfuse
```
What's the output of:
```
brew info --cask osxfuse
```
?
